src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -376,6 +376,66 @@ void jit_floor_emitter::emit_isa(const std::vector<size_t>& in_vec_idxs, const s
 std::set<std::vector<element::Type>> jit_floor_emitter::get_supported_precisions(const std::shared_ptr<ov::Node>& node) {
     return {{element::f32}}; 
 }
+
+/// LOGICAL AND ///
+jit_logical_and_emitter::jit_logical_and_emitter(ov::intel_cpu::riscv64::jit_generator* host, ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                 const std::shared_ptr<ov::Node>& node)
+    : jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {
+    prepare_table();
+}
+
+jit_logical_and_emitter::jit_logical_and_emitter(ov::intel_cpu::riscv64::jit_generator* host, ov::intel_cpu::riscv64::cpu_isa_t host_isa,
+                                 const ov::element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {
+    prepare_table();
+}
+
+size_t jit_logical_and_emitter::get_inputs_num() const {
+    return 2;
+}
+
+size_t jit_logical_and_emitter::aux_gprs_count() const {
+    return 1;
```

Comment:
We use 1 gpr for data table (to [store address](https://github.com/openvinotoolkit/openvino/blob/a37be6547cc02e6e211636a7c83a53b82ca3bf97/src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_emitter.cpp#L51) of the table) and also your implementation needs 1 gpr to store `one` value. 

It means that `aux_gprs_count()` should return `2`. Could you please fix it?

Reply:
done